graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/ConditionModel.java

```diff
@@ -0,0 +1,15 @@
+package io.stargate.graphql.schema.graphqlfirst.processor;
+
+import io.stargate.db.query.Predicate;
+import io.stargate.db.query.builder.BuiltCondition;
+
+public interface ConditionModel {
```

Comment:
Why extract that interface?
`WhereConditionModel` and `IfConditionModel` are 100% identical, we could use a single class.